scripts/gui/PauseMenu.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using Godot;
+
+public class PauseMenu : Control
+{
+    public void ResumePressed()
+    {
+        GUICommon.Instance.PlayButtonPressSound();
+        GetTree().Paused = false;
```

Comment:
This causes conflict with the pause button (the small circular one next to the menu button), so if I click resume on the menu while the pause button is still on paused mode, the game unpauses but the button is still active (play icon). Now I'm wondering if the pause button should be removed. @hhyyrylainen Should it just be removed? It's kind of redundant if the player could just pause the game by opening up the pause menu.

Reply:
I've seen let's plays where players press it. Also it is kinda needed for: https://github.com/Revolutionary-Games/Thrive/issues/192

Reply:
Ah, alright then.

Reply:
So instead they need to use some method to collaborate on when the game should be paused. Perhaps a singleton PauseManager or something could be created to handle that.

Reply:
Yeah, that's that's what I thought.